examples/nas/legacy/textnas/README_zh_CN.md

```diff
@@ -1,6 +1,6 @@
 # TextNAS: A Neural Architecture Search Space tailored for Text Representation
 
-TextNAS 由 MSRA 提出 正式版本。
+TextNAS 由 MSRA 提出 正式版本。 Official Release.
```

Comment:
Translate it.

Reply:
I don't know why some translated sentences are repeated twice, sometimes in English, sometimes in Chinese. So I had to manually delete the repeated sentences......

Reply:
search it in crowdin, but do not to directly update the output. It will be overwritten next time.

Reply:
okay~ crowdin will be modified synchronously